src/monarch_ingest/cli_utils.py

```diff
@@ -487,32 +488,30 @@ def do_release(dir: str = OUTPUT_DIR, kghub: bool = False):
         )
 
         # copy data to monarch-archive bucket only, so that we don't keep so many copies of the same huge files
-        sh.gsutil(*f"-q -m cp data gs://monarch-archive/monarch-kg-dev/{release_ver}/".split(" "))
+        sh.gsutil(*f"-q -m cp -r data/* gs://monarch-archive/monarch-kg-dev/{release_ver}/data".split(" "))
 
         # index and upload to kghub s3 bucket
         if kghub:
-            kghub_release_ver = release_ver.replace("-", "")
-            sh.mkdir("-p", f"{dir}/stats")
-            sh.mv(f"{dir}/merged_graph_stats.yaml", f"{dir}/stats")
+            kghub_release_ver = str(release_ver).replace("-", "")
+            logger.info(f"Uploading to kghub: {kghub_release_ver}...")
+
+            kg_hub_files = f"{dir}/monarch-kg.tar.gz {dir}/rdf/ {dir}/merged_graph_stats.yaml"
+            sh.gsutil(
+                *f"-q -m cp -r -a public-read {kg_hub_files} s3://kg-hub-public-data/kg-monarch/{kghub_release_ver}".split(" ")
+            )
+            sh.gsutil(*f"-q -m cp -r -a public-read {kg_hub_files} s3://kg-hub-public-data/kg-monarch/current".split(" "))
             sh.multi_indexer(
                 *f"-v --directory {dir} --prefix https://kg-hub.berkeleybop.io/kg-monarch/{kghub_release_ver} -x -u".split(
```

Comment:
Please change `https://kg-hub.berkeleybop.io/` to `https://kghub.io/` - trying to use the shorter prefix more consistently.